pages/linux/steghide.md

```diff
@@ -23,6 +23,6 @@
 
 `steghide encinfo`
 
-- Embed data in a JPEG image, encrypted with Blowfish in ECB mode:
```

Comment:
I liked this one more. @sbrl What about you?

Reply:
@zdroid @sbrl And this way ?...

Reply:
I think the new one is better. Note that you might need to go to the "Files changed" bit to see the latest version @zdroid, since GH's change viewing is being weird.

Reply:
I did see it. It was added after this comment, but I felt no need to change my approval. :smile: